pom.xml

```diff
@@ -32,7 +32,7 @@
         <main.basedir>${project.basedir}</main.basedir>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <jdk.version>8</jdk.version>
+        <jdk.version>17</jdk.version>
```

Comment:
if were going to update the minimum version, I think we should probably do the rest of the changes I suggested in https://github.com/hazelcast/hazelcast-remote-controller/pull/69 as well.

Specifically:
- Use `release` over `source` / `target`options
- Use a version of `maven-compiler-plugin` released in this decade

Reply:
added in https://github.com/hazelcast/hazelcast-remote-controller/pull/72/commits/49105e8328d338831b1c44a9db311182661b3a1a